cli.ts

```diff
@@ -26,136 +27,194 @@ Usage:
     aleph <command> [...options]
 
 Commands:
-    ${Object.entries(commands).map(([name, desc]) => `${name.padEnd(15)}${desc}`)
-    .join('\n    ')
-  }
+    ${
+  Object.entries(commands).map(([name, desc]) => `${name.padEnd(15)}${desc}`)
+    .join("\n    ")
+}
 
 Options:
     -v, --version  Prints version number
     -h, --help     Prints help message
-`
+`;
 
 async function main() {
-  const { _: args, ...options } = parse(Deno.args)
+  const { _: args, ...options } = parse(Deno.args);
+
+  // check deno version
+  const [major, minor, patch] = minDenoVersion.split(".").map((p) => parseInt(p));
+  const [currentMajor, currentMinor, currentPatch] = Deno.version.deno.split(
+    ".",
+  ).map((p) => parseInt(p));
+  if (
+    currentMajor < major || (currentMajor === major && currentMinor < minor) ||
+    (currentMajor === major && currentMinor === minor && currentPatch < patch)
+  ) {
+    log.fatal(`Aleph.js requires Deno v${minDenoVersion} or later`);
+  }
```

Comment:
Consider using a `semver` library to do this (`x/semver` for example). Parsing the version like this may break in the future (and probably already breaks when using canary releases).